ssl/s3_lib.c

```diff
@@ -3545,20 +3545,23 @@ long ssl3_ctx_callback_ctrl(SSL_CTX *ctx, int cmd, void (*fp) (void))
     return (1);
 }
 
+const SSL_CIPHER *ssl3_get_cipher_by_id(uint32_t id)
+{
+    SSL_CIPHER c;
+
+    c.id = id;
+    return OBJ_bsearch_ssl_cipher_id(&c, ssl3_ciphers, SSL3_NUM_CIPHERS);
```

Comment:
Was that supposed to say *change* the names? Maybe, although there are a whole bunch of "ssl3_*" type things (and "tls1_*" things for that matter) which are badly named. Could be a whole lot of work. Not opposed though if someone wants to take it on ;-)